nl/guides/exercises/creating-exercises/content/index.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Oefeningen opstellen: leesactiviteit"
+sidebarTitle: Leesactiviteit
+order: 4
+prev: true
+next: true
+---
+
+
+# Oefeningen opstellen: leesactiviteit
+
+Als je in de vorige stap van de handleiding de _template repository_ gebruikt hebt, bevat je repository al een oefening en leesactiviteit.
+Hier zullen we nu zelf nog een leesactiviteit toevoegen.
+Dit is handig om extra informatie of kleine stukken theorie te integreren in je leerpad.
+De bestaande leesactiviteit kan uiteraard als voorbeeld dienen.
+
+{: .callout.callout-success}
+> #### Een leesactiviteit lijkt goed op een oefening
+> De structuur en `config.json` voor een leesactiviteit zijn identiek aan deze van een oefening.
+> Er zijn echter twee grote verschillen: de waarde van `type` moet ingesteld worden op `content` en alles die specifiek voor oefeningen is, mag achterwege gelaten worden.
```

Comment:
Kramdown extensions are not supported in Vitepress, so this page doesn't compile. I would transform it to something like this:

```suggestion
::: info Analoog aan een oefening
De structuur en `config.json` voor een leesactiviteit zijn identiek aan deze van een oefening.
Er zijn echter twee grote verschillen: de waarde van `type` moet ingesteld worden op `content` en alles die specifiek voor oefeningen is, mag achterwege gelaten worden.
:::
```